exercises/kindergarten-garden/description.md

```diff
@@ -23,7 +23,7 @@ There are 12 children in the class:
 - Eve, Fred, Ginny, Harriet,
 - Ileana, Joseph, Kincaid, and Larry.
 
-Each child gets 4 cups, two on each row. The children are assigned to
```

Comment:
Removing this 'to' changes the meaning of the sentence.
It probably needs a more comprehensive rewording.

Reply:
Are the children alphabetized or the cups?  If it is the cups then I didn't pay close enough attention to how the exercise is intended to operate.   If it is referring to the children then the `to` should be dropped.  I'll look this over again as well.